### ISSUE TITLE
Audio muting for Amazon.co.uk

### ISSUE THREAD

**Issue by 101w2in** [richardfrost/AdvancedProfanityFilter#365]
Has anyone managed to get the audio muting to work on amazon.co.uk?

I've tried the below, but it doesn't work.

```html
{
  "www.amazon.co.uk": [
    {
      "mode": "element",
      "subtitleSelector": "div.webPlayerContainer  span",
      "tagName": "div"
    }
  ]}
```

**Comment by richardfrost** [comment]
Thanks for your interest! Unfortunately I don't have access to amazon.co.uk, but have you tried using the config that is being used for amazon.com?

```json
{
  "www.amazon.co.uk": [
    {
      "displayHide": "none",
      "displaySelector": "div.webPlayerContainer div.f35bt6a",
      "displayShow": "",
      "iframe": false,
      "mode": "watcher",
      "parentSelector": "div.webPlayerContainer div p > span",
      "subtitleSelector": "div.webPlayerContainer div span > span",
      "videoSelector": "div.webPlayerElement video[src]"
    }
  ]
}
```

Let me know how that goes. If that doesn't work, let me know and lets see if we can work something out.

**Comment by richardfrost** [comment]
This config was included in [version 2.31.0](https://github.com/richardfrost/AdvancedProfanityFilter/releases/tag/2.31.0). Feel free to re-open or create a new issue if you have trouble in the future.
